CHANGELOG.md

```diff
@@ -1,3 +1,125 @@
+# v1.38.1
+
+_Notice: The v1.38.0 images were promoted incorrectly due to a process error. Do not use any images from `v1.38.0` and upgrade directly to `v1.38.1`._
+
+## Changes by Kind
+
+### Urgent Upgrade Notes
+*(No, really, you MUST read this before you upgrade)*
+
+#### Breaking Metrics Changes
+
+Node plugin metrics have been renamed to follow Prometheus best practices:
+- Added `aws_ebs_csi_` prefix
+- Added `_total` suffix for counters
+- Changed time units from microseconds to seconds for all counters
+
+The controller plugin metrics now use the prefix `aws_ebs_csi_` instead of `cloudprovider_aws_`. The old metric names can be disabled via the CLI parameter `--deprecated-metrics=false` on the controller. This will default to `true` in a future version of the EBS CSI Driver. The old metric names (`cloudprovider_aws_*`) are deprecated and will be removed in a future version of the EBS CSI Driver.
+
+#### [ACTION REQUIRED] Update to the EBS CSI Driver IAM Policy
+
+_(This warning is the same as previous releases and can be disregarded if you have already taken appropriate action)_
+
+Due to an upcoming change in handling of IAM polices for the CreateVolume API when creating a volume from an EBS snapshot, a change to your EBS CSI Driver policy may be needed. For more information and remediation steps, see [GitHub issue #2190](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/2190). This change affects all versions of the EBS CSI Driver and action may be required even on clusters where the driver is not upgraded.
+
+### Feature
+
+- Confirm metrics to Prometheus best practices ([#2248](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2248), [@torredil](https://github.com/torredil))
+- Enable the `VolumeAttributesClass` by default for the Kustomize deployment. If you are deploying using the Kustomize manifests on a cluster that does not have the `VolumeAttributesClass` feature gate enabled on the control plane, you may see harmless extra failures related to the feature in the csi-provisioner and/or csi-resizer logs. ([#2240](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2240), [@ConnorJC3](https://github.com/ConnorJC3))
```

Comment:
Nit: Can we make a Kustomize section or something,  I wouldn't call this a feature.

Reply:
The sections are auto-generated by the script, I'd rather not change them unless you feel super strongly for consistency with future versions.